data/the emerald sword.txt

```diff
@@ -0,0 +1,803 @@
+			`	"They have kept the artifact for themselves to sell on the Black market, and their base of operations is right here in Algieba."`
+			`	The hologram zooms further in, on Watcher, and they point at it.`
+			`	"You will find them in this uninhabited moon. They are hiding out in this forest, and the artifact is with them. Expect hostilities."`
+			`	You thank Mammon for their services and get up to leave.`
+			`	"Farewell, captain <last>. Remember: I am always watching."`
+				accept
+	
+	on defer
+		set visited
+	
+	on accept
+		set "Paid Mammon"
+		fail "The Box 3"
+
+mission "Mammon Payment"        
+	name "Mammon Payment"        
+	landing
+	invisible
+
+	to offer
+		has "Paid Mammon"
+		credits >= 10000000
+
+	on offer
+		payment -10000000
+		conversation     
+			`You receive a message informing you of a very large transaction: 10 million credits. Mammon's crew have withdrawn the money you paid for the information`    
+				decline
+
+mission "The Box 5"
+	landing
+	name "The box"
+	description "Destroy the mercenary fleet."
+	source "Watcher"
+	destination "Watcher"
+	
+	to offer
+		has "The Box 3: offered"
+	
+	on offer
+		conversation
+			`You arrive on the uncolonized world, and scout around the area Mammon pointed out. It's relatively easy to find some smoke rising from a campfire in a clearing, but as you approach to investigate, you're surprised to find some large ships taking off to greet you, ditching the green tarps used to hide among the trees. Looks like they really don't want you here. It's a fight!`
+				launch
+	
+	npc disable
+		government "Bounty Hunter"
+		personality unconstrained launching nemesis heroic plunders harvests target marked
+			confusion 20
+		system "Algieba"
+		fleet "Marauder fleet X"
+		dialog
+			`You managed to destroy the entire mercenary fleet. You should have no problem landing on their camp now and recovering the cube.`
+	
+	on visit
+		conversation
+			`You attempt to evade the mercenary ships and make a quick assault on their encampment, but the ships follow you through the atmosphere, and ground personnel start shooting at you. Bombing it on a pass means you risk destroying the cube. You have no choice but to go back to space and eliminate the mercenary fleet.`
+				launch
+	
+mission "The Box 6a"
+	landing
+	name "The box"
+	source "Watcher"
+	destination "Vinci"
+	invisible
+	
+	to offer
+		has "The Box 5: done"
+	
+	on offer
+		conversation
+			`You come in for a landing, and the few mercenaries that stayed on the ground march to your ship carrying a makeshift white flag. Dozens of men and women, tough looking and in uniform, carrying weapons but remaining peaceful and organized. These weren't ordinary criminals, it seems.`
+			`	"We know when we lost" One of them yells to your ship on a megaphone. "What is it that you want?"`
+			choice
+				`(Shoot them)`
+					goto shoot
+				`(Talk to them)`
+			
+			`	You land and come out with your crew, forming your own defensive line.`
```

Comment:
The player might not have a crew at this point (might be flying a smaller ship alone).

Reply:
I considered it cafully, but 1, there is no way to check whether the player has a crew or not, and 2, it's slightly unlikely that the player will be flying solo in a single-person ship at this point, since the last mission has you defeat an entire fleet of pirates.
The player could be piloting a kor automaton, but even those take crew if you want to use turrets.